common/src/main/resources/assets/gtceu/compass/pages/en_us/tools/scythe.xml

```diff
@@ -0,0 +1,17 @@
+<page>
+    <h1>
+        Scythe
+    </h1>
+    <h3>
+        Source
+    </h3>
+    <text><style color="0xaaaaaa" italic="true">Author: <style color="0xa4e7ff" hover-info="TacoMonkey's Github" underlined="true" url-link="https://github.com/tacomonkey11">TacoMonkey</style><br/></style></text>
+    <text>
+        A sword-like weapon which currently has no use. Item functionality is Not Yet Implemented.
```

Comment:
you could mention here that it's supposed to break & replant crops in a 3x3, because I believe that is the use for it?